src/Auth0Client.ts

```diff
@@ -309,7 +309,7 @@ export class Auth0Client {
     this.customOptions = getCustomInitialOptions(options);
 
     this.useRefreshTokensFallback =
-      this.options.useRefreshTokensFallback !== false;
+      this.options.useRefreshTokensFallback === true;
```

Comment:
I think we can drop `this.useRefreshTokensFallback` and rely on `this.options.useRefreshTokensFallback`.

We used this separate field because previously `undefined` led to `true`, while now it's `false` and can remain undefined.

Reply:
Now only uses `this.options.useRefreshTokensFallback` throughout